### PR TITLE
The feels-like-temperature CLI and associated bats tests.

### PR DESCRIPTION
**Pull request by Katie-Howard**
Addresses #GitHubissuenum

Description

Testing:
 - [X] Ran tests and they passed OK
 - [X] Added new tests for the new feature(s)

### CODE REVIEW

File: bin/improver-feels-like-temp [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    """ Load in the arguments for feels like temperature and ensure they are
+    set correctly. Then calculate the feels like temperature using the data
+    in the input cubes."""
+    parser = ArgParser(
+        description="This calculates the feels like temperature using a "
+                    "combination of the wind chill index and Steadman's "
+                    "apparent temperature equation.")
+    parser.add_argument("temperature", metavar="TEMPERATURE",
+                        help="Path to a NetCDF file of air temperatures at "
+                        "the surface.")
```
[the remark this answers is not in the document]
Comment (Katie-Howard) [reply bin/improver-feels-like-temp]: Changed.

File: bin/improver-feels-like-temp [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    parser.add_argument("wind_speed", metavar="WIND_SPEED",
+                        help="Path to the NetCDF file of wind speed at the "
+                        "surface.")
```
[the remark this answers is not in the document]
Comment (Katie-Howard) [reply bin/improver-feels-like-temp]: Done.

File: bin/improver-feels-like-temp [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    parser.add_argument("relative_humidity", metavar="RELATIVE_HUMIDITY",
+                        help="Path to the NetCDF file of relative humidity "
+                        "at the surface.")
```
[the remark this answers is not in the document]
Comment (Katie-Howard) [reply bin/improver-feels-like-temp]: Changed.

File: bin/improver-feels-like-temp [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    parser.add_argument("pressure", metavar="PRESSURE",
+                        help="Path to a NetCDF file of air pressures at "
+                        "the surface.")
```
[the remark this answers is not in the document]
Comment (Katie-Howard) [reply bin/improver-feels-like-temp]: Changed.

**Comment by Katie-Howard** [comment]
@carolinejjones I have made those changes that you requested.

**Comment by carolinejjones** [comment]
Thanks but I'm afraid the 01-help.bat will need to be updated too.

**Comment by Katie-Howard** [comment]
I've updates 01-help.bats with the updated descriptions now too.

**Pull request merged**